.vscode/settings.json

```diff
@@ -1,6 +1,3 @@
 {
-  "discord.enabled": false,
-  "discord.lowerDetailsEditing": "",
-  "discord.lowerDetailsDebugging": "",
   "deno.enable": false
```

Comment:
this file shoudn't be needed.

Reply:
Fair enough. Thanks for the comment! Deleted.